packages/cdk8s-cli/templates/typescript-app/main.test.ts

```diff
@@ -0,0 +1,11 @@
+import {MyChart} from './main';
+import {App, Testing} from "cdk8s";
+
+describe('Placeholder', () => {
+    test('Empty', () => {
+        const app = new App();
+        const chart = new MyChart(app, 'test-chart');
+        const results = Testing.synth(chart)
+        expect(results).toHaveLength(0);
```

Comment:
I would show how to use snapshot tests:

```ts
expect(Testing.synth(chart)).toMatchSnapshot();
```

Reply:
changed

Reply:
I updated this to have the .toMatchSnapshot() call but it's causing the build to fail. 

When the `init` happens it also builds and run tests. Those tests then fail because the `jest -u` wasn't passed, the snapshot won't be created unless specifically told to update (this doesn't appear to be a problem locally, though). Looking through the code it appears that after the project is scaff'd it'll do a 'build'. Before I go and rework the scaff so that it calls a special 'first time build' (with the -u), is there a better way to handle this IYO?

Reply:
Alright, I just replaced the 'npm run build' with a breakout of the individual commands and things are passing now.